<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SVDBSupplySystemEditWidget</class>
 <widget class="QWidget" name="SVDBSupplySystemEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>446</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="labelDisplayName">
       <property name="minimumSize">
        <size>
         <width>60</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QtExt::LanguageStringEditWidget1" name="lineEditName"/>
     </item>
     <item>
      <widget class="QtExt::ColorButton" name="pushButtonColor">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="title">
      <string/>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Supply type:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBoxSupplyType">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidgetSupply">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="currentIndex">
         <number>3</number>
        </property>
        <widget class="QWidget" name="pageStandAlone">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Maximum mass flux:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" rowspan="2">
           <widget class="QtExt::ValidatingLineEdit" name="lineEditMaxMassFlux"/>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>kg/s</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2" rowspan="2">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>C</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Supply temperature:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QtExt::ValidatingLineEdit" name="lineEditSupplyTemp"/>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>252</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageSubNetwork">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLineEdit" name="lineEditSubNetwork"/>
            </item>
            <item>
             <widget class="QToolButton" name="toolButtonSubNetwork">
              <property name="text">
               <string>...</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>314</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="pageDatabaseFMU"/>
        <widget class="QWidget" name="pageUserDefinedFMU">
         <layout class="QGridLayout" name="gridLayout">
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Supply FMU:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QtExt::BrowseFilenameWidget" name="widgetBrowseFileNameSupplyFMU" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Maximum mass flux FMU:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QtExt::ValidatingLineEdit" name="lineEditMaxMassFluxFMU"/>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>kg/s</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Heating power FMU:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QtExt::ValidatingLineEdit" name="lineEditHeatingPowerFMU"/>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>kW</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>229</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QtExt::LanguageStringEditWidget1</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_LanguageStringEditWidget1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QtExt::ColorButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtExt_ColorButton.h</header>
   <slots>
    <signal>colorChanged()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtExt::ValidatingLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">QtExt_ValidatingLineEdit.h</header>
   <slots>
    <signal>editingFinishedSuccessfully()</signal>
   </slots>
  </customwidget>
  <customwidget>
   <class>QtExt::BrowseFilenameWidget</class>
   <extends>QWidget</extends>
   <header>QtExt_BrowseFilenameWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
